Makefile

```diff
@@ -70,6 +70,21 @@ clean:
 clean_pkg:
 	rm -rf ../../../../pkg Godeps/_workspace/pkg
 
+# Remove everything including stuff pulled down by bootstrap.sh
+cleanall:
+	# symlinks
+	for f in config data py-vtdb; do test -L ../../../../$$f && rm ../../../../$$f; done
+	# directories created by bootstrap.sh
+	# - exclude vtdataroot and vthook as they may have data we want
+	rm -rf ../../../../bin ../../../../dist ../../../../lib ../../../../pkg
+	# keep the vendor.json file but nothing else under the vendor directory as it's not actually part of the Vitess repo
+	rm -rf vendor/cloud.google.com vendor/github.com vendor/golang.org vendor/google.golang.org vendor/gopkg.in
+	# other stuff in the go hierarchy that is not under vendor/
+	rm -rf ../../../golang.org ../../../honnef.co
```

Comment:
I'm not removing anything that wasn't created by `bootstrap.sh`.  If someone has added local changes I agree they'll lose them but isn't something like `make cleanall` intended to do what it says?

Reply:
If you want me to be very explicit and check each and every package that's pulled down by `bootstrap.sh` then I can do that. It's not clear to me what you're trying to prevent or if you just want to be extremely cautious when running such a command.

Reply:
I was more worried about such external dependencies being used by other projects. But I guess such people can stay away from using cleanall.